src/NuGet.Core/NuGet.ProjectModel/Strings.resx

```diff
@@ -132,4 +132,7 @@
   <data name="Log_InvalidImportFramework" xml:space="preserve">
     <value>Imports contains an invalid framework: '{0}' in '{1}'.</value>
   </data>
+  <data name="MissingVersionProperty" xml:space="preserve">
+    <value>Package dependencies must specify a version range.</value>
```

Comment:
Can we point to the actual line where this is missing. This could get tricky in a big file, at a minimum specify the name, better specify the line and column as well. This could also become later a way to auto fix (double click in the issues list, and go right to the line in project.json)